src/main/resources/headers/okapi-token.yaml

```diff
@@ -0,0 +1,7 @@
+in: header
+name: X-Okapi-Token
+description: Okapi Token
+# The /_/tenant route will not have this header.
+required: false
```

Comment:
`required: false` means optional.
If some routes _require_ the token an okapi-token-required.yaml with `required: true` should be used.
If `/_/tenant` doesn't use a Token at all the okapi-token.yaml should be omitted.

Reply:
Good catch. Yes agree.

Reply:
See below for comment on how I propose we handle the dependency for `/_/tenant`.  But one side effect of this is that we'll need to leave this `required: false`. The other module that I'm aware of that uses folio-vertx-lib [also left it as](https://github.com/folio-org/mod-eusage-reports/blob/8cdf125331cd6b6fd31428d4b283062b4b1d7f1b/src/main/resources/openapi/headers/okapi-tenant.yaml#L4) `required: false` probably for this reason.